src/api/js/Makefile

```diff
@@ -0,0 +1,11 @@
+# Set to 1 to activate API logging
```

Comment:
This seems to be something to fold into the build generators (python/cmake) that produce the make files or other inputs to the compilers.